features/dd-sdk-android-rum/src/main/kotlin/com/datadog/android/rum/internal/instrumentation/gestures/GesturesListener.kt

```
@@ -53,7 +53,7 @@ internal class GesturesListener(
     override fun onSingleTapUp(e: MotionEvent): Boolean {
         val decorView = windowReference.get()?.decorView
         handleTapUp(decorView, e)
-        return false
+        return true
```

are you sure this has no impact on the customer side? normally we shouldn't consume events, but just read them and let them propagate further. And it is customer code to decide where it should be consumed.

---

If I understand correctly, we use `WindowCallbackWrapper` as the interceptor of touch event, and indeed in `WindowCallbackWrapper` we always return false to let event propagate. 

`GesturesListner` derives from  `WindowCallbackWrapper` to handle the touch event, and its return value is not taken into account the final return value of `WindowCallbackWrapper`, so I don't think it will affect user behavior.

But since it's a quite impactful change, it will be very helpful if you can double check in the code if what I am saying is correct.

Also cc @mariusc83